metadata/deploy-tenant.sh

```
@@ -1,3 +1,4 @@
+# deploy-tenant.sh
 #!/usr/bin/env bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)